blob/driver/driver.go

```diff
@@ -61,6 +61,9 @@ type WriterOptions struct {
 	// write in a single request, if supported. Larger objects will be split into
 	// multiple requests.
 	BufferSize int
+	// MaxUploadParts specifies the maximum number of uploaded parts.
```

Comment:
You will also need to add this option to WriterOptions as defined in blob.go, and copy the value to driver.WriterOptions. Search for MaxConcurrency in blob.go.